addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -95,7 +95,7 @@ var SnippetEditor = Widget.extend({
                         $clone.appendTo(this.$body).removeClass('d-none');
                         return $clone;
                     },
-                    start: this._onDragAndDropStart.bind(this),
+                    start: (...args) => this._onDragAndDropStart(...args),
```

Comment:
I don't think this change changes anything? :thinking: And if I am right, I even prefer the version with `bind` so... :)